src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/expectedInputJavadocComment.txt

```diff
@@ -1,25 +1,25 @@
-JAVADOC -> <p>\r\nMy {@code Class}\r\n</p>\r\n@see never<EOF> [0:0]
-|--HTML_ELEMENT -> <p>\r\nMy {@code Class}\r\n</p> [0:0]
-|   `--PARAGRAPH -> <p>\r\nMy {@code Class}\r\n</p> [0:0]
-|       |--P_TAG_OPEN -> <p> [0:0]
-|       |   |--OPEN -> < [0:0]
-|       |   |--P_HTML_TAG_NAME -> p [0:1]
-|       |   `--CLOSE -> > [0:2]
-|       |--NEWLINE -> \r\n [0:3]
+JAVADOC -> <p>\nMy {@code Class}\n</p>\n@see never<EOF> [0:3]
+|--HTML_ELEMENT -> <p>\nMy {@code Class}\n</p> [0:3]
+|   `--PARAGRAPH -> <p>\nMy {@code Class}\n</p> [0:3]
+|       |--P_TAG_OPEN -> <p> [0:3]
+|       |   |--OPEN -> < [0:3]
+|       |   |--P_HTML_TAG_NAME -> p [0:4]
+|       |   `--CLOSE -> > [0:5]
+|       |--NEWLINE -> \n [0:6]
 |       |--TEXT -> My  [1:0]
 |       |--JAVADOC_INLINE_TAG -> {@code Class} [1:3]
 |       |   |--JAVADOC_INLINE_TAG_START -> { [1:3]
 |       |   |--CODE_LITERAL -> @code [1:4]
 |       |   |--WS ->   [1:9]
 |       |   |--TEXT -> Class [1:10]
 |       |   `--JAVADOC_INLINE_TAG_END -> } [1:16]
-|       |--NEWLINE -> \r\n [1:17]
```

Comment:
Test passes because it not only replaces test `\r\n` to `\n` in input, but also replaces it in output. It fixes the files on both sides so they can compared successfully on any OS.
@kazachka probably copied expected output after running test to file not realizing there was this small difference.